StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -466,6 +468,18 @@ extension FinancialConnectionsAPIClient: FinancialConnectionsAPI {
         )
     }
 
+    func selectInstitution(clientSecret: String, institutionId: String) -> Promise<FinancialConnectionsSelectInstitution> {
+        let body: [String: Any] = [
+            "client_secret": clientSecret,
+            "currently_selected_institution": institutionId,
+        ]
+        return self.post(
+            resource: APIEndpointInstitutionSelected,
+            parameters: body,
+            useConsumerPublishableKeyIfNeeded: true
```

Comment:
We should never have a consumer publishable key available here, so this could be `false` as well, but I guess it doesn’t matter.